extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -483,12 +483,15 @@ export class AmpA4A extends AMP.BaseElement {
     }
     // If in localDev `type=fake` Ad specifies `force3p`, it will be forced
     // to go via 3p.
-    if (getMode().localDev &&
-        this.element.getAttribute('type') == 'fake' &&
-        this.element.getAttribute('force3p') == 'true') {
-      this.adUrl_ = this.getAdUrl();
-      this.adPromise_ = Promise.resolve();
-      return;
+    if (getMode().localDev && this.element.getAttribute('type') == 'fake') {
```

Comment:
Happy to remove, but note that this check was here previously. Was trying to make as little change as possible...